### ISSUE TITLE
Multiple uses of `<main>` in themes

### ISSUE THREAD

**Issue by lancegliser** [FusionAuth/fusionauth-issues#2029]
### Description 
As I understand it, the purpose of the html `<main/>` tag is pretty specific. You _can_ have more than one and it will compile, similar to `<h1 />`. For semantic and accessibility issues it's [discouraged](https://stackoverflow.com/questions/34896476/can-i-use-more-than-one-main-html-tag-in-the-same-page). Maybe this should be corrected?

### Affects versions
Fusion auth hosted instance:  version 1.41.3

### Steps to reproduce
Review the template helpers file and related css.

### Expected behavior
`main` is used only for the primary content.

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).
